duradmin/src/main/webapp/js/spaces-manager.js

```diff
@@ -3288,6 +3290,16 @@ $(function() {
       });
     },
 
+   _isS3 : function(storeId) {
```

Comment:
Not a functional concern, but perhaps this should sit next to the _isSnapshot function?